meson.build

```diff
@@ -299,6 +299,9 @@ if build_daemon
 endif
 libm = cc.find_library('m', required: false)
 zlib = dependency('zlib')
+if get_option('hotplug_netlink_kernel')
```

Comment:
Does this need a new option? Can't you just go off systemd?

Reply:
I could probably re-use HAVE_SYSTEMD too...